### ISSUE TITLE
Cropping is ignored

### ISSUE THREAD

**Issue by d07RiV** [udevbe/tinyh264#8]
H264 stream contains cropping parameters for frames with sizes that are not factors of 16 (such as 1920x1080). The C decoder has a function "h264bsdCroppingParams" to get these parameters, but this library ignores it, and decodes the frame as 1920x1088 instead.

**Comment by Zubnix** [comment]
Yes that would be good to have. I currently crop the resulting frame using webgl texture coordinates. Unfortunately I don't really have the skill or time to quickly implement this change. I'd be happy to accept a PR!
